src/llmcompressor/transformers/finetune/session_mixin.py

```diff
@@ -222,7 +223,9 @@ def create_optimizer(self):
                 len(self.train_dataset) / total_batch_size
             )
 
-        initialize(optimizer=self.optimizer, steps_per_epoch=self.total_steps_per_epoch)
+        update_state(
```

Comment:
How do you feel about just calling 

`active_session().update_state(**kwargs)`

As opposed to adding another session_function?
I think we should aim to remove the functions in that file.

Reply:
I agree for now, we can talk about the high level design at a later time.